examples/cfd/04_burgers.ipynb

```diff
@@ -252,9 +257,9 @@
     "op = Operator([update_u, update_v] + bc_u + bc_v)\n",
     "\n",
     "# Execute the operator for a number of timesteps\n",
-    "op(time=nt, dt=dt, a=nu)\n",
-    "\n",
-    "plot_field(u.data[0])"
+    "for iters in range(batches):\n",
+    "    op(time=nt, dt=dt, a=nu)\n",
```

Comment:
I would add `time_m=iters*batch_size, time_M=(iters+1)*batches -1` as this has a lot of assumtions from 1st order in time but would be different for different orders.

Reply:
Mathias I do not understand well what you mean, can you add here 2-3 lines of code?

Reply:
so mainly:
`iters=0` runs from `t=0` to `t=nt`
`iters=1` runs from `t=nt` to `t=nt + nt`
`iters=2` runs from `t=2*nt` to `t=2*nt + nt`
...

Which doesn't really show here

Reply:
You said earlier that `nt = batches*batch_size` so I guess you mean:
`batches=0` runs from `t=0` to `t=batch_size`
`batches=1` runs from `t=batch_size` to `t=batch_size + batch_size`
`batches=2` runs from `t=2*batch_size` to `t=2*batch_size + batch_size`

Up to `batches*batch_size`

Reply:
yes